packages/common/src/events/edit.ts

```diff
@@ -96,6 +95,7 @@ export async function updateHubEvent(
     editGroups: model.editGroups,
     endDate: model.endDate,
     endTime: model.endTime,
+    inPersonCapacity: model.inPersonCapacity,
```

Comment:
We may want to stub out default values for the `inPersonCapacity` in the `packages/common/src/events/defaults.ts` module's `buildDefaultEventEntity` and `buildDefaultEventRecord` methods.